calculator/src/test/java/edu/nextstep/camp/calculator/CalculatorViewModelTest.kt

```diff
@@ -0,0 +1,76 @@
+package edu.nextstep.camp.calculator
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import com.google.common.truth.Truth.assertThat
+import edu.nextstep.camp.calculator.domain.Calculator
+import edu.nextstep.camp.calculator.domain.Operator
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+
+@ExtendWith(InstantExecutorExtension::class)
+internal class CalculatorViewModelTest {
+
+    private lateinit var calculatorViewModel: CalculatorViewModel
+    private lateinit var calculator: Calculator
+
+    @BeforeEach
+    internal fun setUp() {
```

Comment:
```suggestion
   fun setUp() {
```
클래스단에 internal 접근자가 있다면, 함수에는 internal을 붙이지 않아도 됩니다 😅 (이하 관련 내용 동일)